### ISSUE TITLE
Applying the ColorJitter transform

### ISSUE THREAD

**Issue by rickgroen** [nianticlabs/monodepth2#121]
First off, thanks for making this project publicly available.

I was wondering about the use of ColorJitter in datasets/mono_dataset.py (lines 109 & 175-176)
https://github.com/nianticlabs/monodepth2/blob/d1c5f03c38305cae4e68917e472d2f9d4eda0b98/datasets/mono_dataset.py#L175-L177

From looking at the PyTorch [source code](https://pytorch.org/docs/stable/_modules/torchvision/transforms/transforms.html#ColorJitter) I see that whenever we call the transform, **get_params()** gets called as well. 
I could be misinterpreting this, but it seems like a new random transformation is applied to each image. This is different in MonoDepth v1, where the same colour augmentation is applied to each image pair.

I am currently choosing between either myself, but wanted to know what you experience in this matter was. Am I correct in that we now apply a new transform to each image (and thus not the same transform for an image pair)? And if so, do you think it makes the predictions of the model more robust against noise?

**Comment by daniyar-niantic** [comment]
In monodepth v1 the augmentation is applied to both images, but only single augmented image is used during training. In both monodepth versions only a single augmented image is used as input to the network. Only original images are used for computing reprojection loss.

**Comment by rickgroen** [comment]
I see. I thought augmented image pairs were also used for reprojection. Thanks for clearing that up!
